style.css

```diff
@@ -1,3 +1,313 @@
 * {
   box-sizing: border-box;
+  /*border: 1px solid red;*/
+}
+
+body {
+  margin: 0 auto;
+}
+
+.announcementbar {
+  margin: 0;
+  width: 100%;
+  background: #fa6938;
+  padding: 0.01rem;
+  font-family: 'Source Sans Pro', sans-serif;
+  color: black;
+}
+
+.announcementbar p {
+  margin-left: 5rem;
+  color: white;
+}
+
+#shipping {
+  color: white;
+  background: rgba(0,0,0,0.2);
+  border-radius: 0.5rem;
+  padding: 0.4rem;
+  text-transform: uppercase;
+}
+
+.top-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.site-logo {
+  height: 8rem;
+  width: auto;
+  margin-left: 5rem;
+}
+
+.nav-list {
+}
+
+.accountlinks {
+  font-family: 'Source Sans Pro', sans-serif;
+  color: gray;
+  text-align: right;
+  margin-right: 5rem;
+  padding: 0.2rem;
+  /*text-decoration: none;*/
+}
+
+.signin {
+  text-decoration: none;
+  color: gray;
+}
+
+.container {
+  display: block;
+}
+
+.InputAddOn {
+  float: left;
+}
+
+.InputAddOn-field {
+  flex: 1;
+}
+
+.InputAddOn-item {
+}
+
+.accountsearch {
+  margin-right: 5rem;
+  position: relative;
+}
+
+.accountsearch button {
+  float: left;
+  border-radius: 0.3rem;
+  padding: 0.2rem;
+  margin: 0.2rem;
+}
+
+.nav-list__item {
+  list-style: none;
+  display: inline;
+  padding: 0.5rem;
+  font-family: 'Source Sans Pro', sans-serif;
+  color: black;
+  font-size: 1.5rem;
+  font-weight: lighter;
+}
+
+
+.fa-search {
+  color: gray;
+}
+
+
+.hero-container {
+  display: block;
+  background: #F5F5F5;
+  border-top-style: groove;
+}
+
+.tieselection {
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  list-style: none;
+  width: 80%;
+  padding-left: 0;
+}
+
+.tie img {
+  width: 100%;
+}
+
+.tie {
+  text-decoration: none;
+}
+
+.tiecolor {
+  text-align: center;
+  font-family: 'Source Sans Pro', sans-serif;
+  color: black;
+}
+
+.tagline {
+  display: block;
+  text-align: center;
+  padding: 1rem;
+  font-family: 'Source Sans Pro', sans-serif;
+  font-size: 4rem;
+}
+
+.three {
+  display: flex;
+  width: 80%;
+  justify-content: center;
+  margin: auto;
+  position: relative;
+}
+
+.class {
+  width: 100%;
+}
+
+.three-img {
+  width: 100%;
+  margin: 1rem;
+  position: relative;
+}
+
+.three-img-text {
+  font-family: 'Source Sans Pro', sans-serif;
+  font-size: 1.3rem;
+  font-weight: bold;
+}
+
+.bow-ties {
+  position: absolute;
+  top: 0rem;
+  left: 12rem;
+}
+
+.pocket-squares {
+  position: absolute;
+  top: 0rem;
+  left: 13rem;
+}
+
+.tie-bars {
+  position: absolute;
+  top: 0rem;
+  left: 13rem;
```

Comment:
Pinning things to the right corner using `left` will probably be tough. A general rule of thumb would be when positioning absolute within the box, pin to the closest edge and things can be locked in.